docs/auto-instrumentation-install.md

```diff
@@ -25,31 +25,45 @@ these frameworks often have pre-built instrumentation capabilities already avail
-- When `operator.enabled=true`, the helm chart deploys a
-[default](https://github.com/signalfx/splunk-otel-collector-chart/blob/main/examples/enable-operator-and-auto-instrumentation/rendered_manifests/operator/instrumentation.yaml)
-opentelemetry.io/v1alpha1 Instrumentation object.
-  - The default Instrumentation supports [AlwaysOn Profiling](https://docs.splunk.com/Observability/apm/profiling/intro-profiling.html) when `splunkObservability.profilingEnabled=true`.
-    - These environment variables will be set when auto-instrumenting applications.
-      - SPLUNK_PROFILER_ENABLED="true"
-      - SPLUNK_PROFILER_MEMORY_ENABLED="true"
-    - Example
-      - [Enable always-on profiling](../examples/enable-operator-and-auto-instrumentation/instrumentation/instrumentation-enable-profiling.yaml)
-  - Users can override the specifications of the default deployed instrumentation by setting override values under `operator.instrumentation.spec`.
-    - Examples
-      - [Add custom environment span tag](../examples/enable-operator-and-auto-instrumentation/instrumentation/instrumentation_add_custom_environment_span_tag.yaml)
-      - [Add trace sampler](../examples/enable-operator-and-auto-instrumentation/instrumentation/instrumentation-add-trace-sampler.yaml)
-      - [Enable always-on profiling partially](../examples/enable-operator-and-auto-instrumentation/instrumentation/instrumentation-enable-profiling-partially.yaml)
-- To view a deployed Instrumentation, you can use the command: `kubectl get otelinst {instrumentation_name} -o yaml`.
-- The `deployment.environment` attribute may be set in the exported traces to identify the [APM deployment environment](https://docs.splunk.com/observability/en/apm/set-up-apm/environments.html). There are two ways to set this attribute:
-  - Use the optional `environment` configuration in `values.yaml`.
-  - Use the Instrumentation spec (`operator.instrumentation.spec.env`) with the environment variable `OTEL_RESOURCE_ATTRIBUTES`.
+- **Operator Deployment (Required)**
+  - `operator.enabled`: Set to `true` to enable deploying the operator.
+    - **Required**: This configuration is necessary for the operator's deployment within your cluster.
+
+- **TLS Certificate Management (Required)**
+  - **Using cert-manager (Recommended)**
+    - `certmanager.enabled`: Enable cert-manager by setting to `true`.
+      - **Check Before Enabling**: Ensure cert-manager is not already installed to avoid multiple instances.
+      - **Recommended**: Cert-manager simplifies the management of TLS certificates, automating issuance and renewal.
+
+  - **Alternative Methods**
+    - **Automatically Generate a Self-Signed Certificate with Helm**
+      - `operator.admissionWebhooks.autoGenerateCert.enabled`: Set to `true` to enable Helm to automatically create a self-signed certificate.
+        - **Use Case**: Suitable when cert-manager is not installed or preferred.
+    - **Provide Your Own Certificate**
```

Comment:
Noticed that the operator chart is not reading the contents of the file. I have an [issue](https://github.com/open-telemetry/opentelemetry-helm-charts/issues/1120) and PR open for the operator chart to fix this.

Reply:
I have a customer waiting on this doc. I would like to propose we merge this PR as is and create a new PR for the changes in https://github.com/open-telemetry/opentelemetry-helm-charts/issues/1120

Reply:
go ahead, we can revise these after the upstream pr is merged